www-client/kristall/kristall-0.7.3~git.recipe

```diff
@@ -0,0 +1,68 @@
+SUMMARY="A cross-platform, multi-protocol visual browser for the small-internet"
+DESCRIPTION="Kristall is a browser without support for css/js/wasm or graphical \
+websites. It can display user-styled documents in several formats, \
+including Gemini, HTML, and Markdown provided by a server via Gemini, \
+Gopher, HTTP/HTTPS, and Finger protocols."
+HOMEPAGE="https://kristall.random-projects.net/"
+COPYRIGHT="2020 Felix Queißner"
+LICENSE="GNU GPL v3"
+REVISION="1"
+srcGitRev="b684f94f1af9a19c1a6fd70d72097a13b75e1ca6"
+SOURCE_URI="https://github.com/MasterQ32/kristall/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="ad8dd1fa28702e94d8830fe4ba0908e44417396e7402ebfa974897bfcd8be7dd"
+SOURCE_FILENAME="kristall-$portVersion-tar.gz"
+SOURCE_DIR="kristall-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
```

Comment:
If primary architecture x86_gcc2 is ok with a build, there wouldn't be a reason to enable "SECONDARY_ARCHITECTURES" (didn't check the build) if it's not building for gcc2 please disable the build there (also remove the build for primary x86):
```diff
- ARCHITECTURES="x86_gcc2 x86 x86_64"
+ ARCHITECTURES="!x86_gcc2 x86_64"
```